trl/models/modeling_base.py

```diff
@@ -164,7 +163,9 @@ class and the arguments that are specific to trl models. The kwargs
                     peft_config.base_model_name_or_path, *model_args, **pretrained_kwargs
                 )
 
-                pretrained_model = PeftModel.from_pretrained(pretrained_model, pretrained_model_name_or_path)
+                pretrained_model = PeftModel.from_pretrained(
+                    pretrained_model, pretrained_model_name_or_path, is_trainable=True
```

Comment:
I think `is_trainable` is only relevant when the peft config is different from `LoraConfig` no? 
Also what if a user loads the model just for inference? I think we can leave this change in a future PR and add `is_trainable` as a kwarg on `from_pretained` of `PreTrainedWrapper`

Reply:
Oh, my mistake. I think you're right. I have only considered my use case. 😅
I'll open another PR for the `is_trainable` as a kwarg.